packages/pancake-uikit/src/components/BalanceInput/BalanceInput.tsx

```diff
@@ -6,15 +6,29 @@ import { BalanceInputProps } from "./types";
 const BalanceInput: React.FC<BalanceInputProps> = ({
   value,
   placeholder = "0.0",
-  onChange,
+  onUserInput,
   currencyValue,
   inputProps,
   isWarning = false,
   ...props
 }) => {
+  const handleOnChange = (e: React.FormEvent<HTMLInputElement>) => {
```

Comment:
This should be `React.ChangeEventHandler<HTMLInputElement>`

Reply:
I got errors when i changed to that one

Reply:
I think it is `React.ChangeEvent<HTMLInputElement>`

Reply:
That looks right.